WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -0,0 +1,742 @@
+import Foundation
+import WordPressComAnalytics
+import WordPressComStatsiOS
+import WordPressShared
+
+@objc class PostListViewController : AbstractPostListViewController, UIViewControllerRestoration, PostCardTableViewCellDelegate {
+    
+    static private let postCardTextCellIdentifier = "PostCardTextCellIdentifier"
+    static private let postCardImageCellIdentifier = "PostCardImageCellIdentifier"
+    static private let postCardRestoreCellIdentifier = "PostCardRestoreCellIdentifier"
+    static private let postCardTextCellNibName = "PostCardTextCell"
+    static private let postCardImageCellNibName = "PostCardImageCell"
+    static private let postCardRestoreCellNibName = "RestorePostTableViewCell"
+    static private let postsViewControllerRestorationKey = "PostsViewControllerRestorationKey"
+    static private let statsStoryboardName = "SiteStats"
+    static private let currentPostListStatusFilterKey = "CurrentPostListStatusFilterKey"
+    static private let currentPostAuthorFilterKey = "CurrentPostAuthorFilterKey"
```

Comment:
Note for future PR's (no need to do anything here)

Maybe do some grouping like...

```
private enum CellNibs : String {
    case postCardTextCell = "PostCardTextCell"
    ....
}
```